### PR TITLE
Fix up theme and extensions hero content to match mocks

### LINKED ISSUE #3215: Fix up theme and extension hero content to closer match mocks
Today: 

add-ons_for_firefox

Mocks:

amo_desktop_-_master_sketch

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #3215

* Centers the SVG images better so avoids special casing margins for pseudo centering.
* Updates styles to be closer to the mocks
* Compresses these big SVGs this saves ~30/40% per svg

Before: 

add-ons_for_firefox

After: 

screen shot 2017-09-22 at 19 50 35
screen shot 2017-09-22 at 19 50 39
screen shot 2017-09-22 at 19 50 47

screen shot 2017-09-22 at 19 51 02

screen shot 2017-09-22 at 19 51 07

screen shot 2017-09-22 at 19 51 14
screen shot 2017-09-22 at 19 50 51
screen shot 2017-09-22 at 19 51 20

### CODE REVIEW

File: src/amo/components/LandingPage/styles.scss [hunk 1]
```diff
@@ -36,28 +29,34 @@
 }
 
 .LandingPage-icon {
-  height: 89px;
-  width: 225px;
+  width: 100%;
```
Comment (tofumatt) [inline src/amo/components/LandingPage/styles.scss]: Damn, I thought my stylelint forced alphabetisation of attributes 😆

File: src/amo/components/LandingPage/styles.scss [hunk 2]
```diff
@@ -36,28 +29,34 @@
...
-    width: 450px;
+    height: 248px;
+    width: 473px;
   }
 }
 
 .LandingPage-addonType-name {
   font-size: $font-size-l;
-  margin: 0;
+  font-weight: lighter;
```
Comment (tofumatt) [inline src/amo/components/LandingPage/styles.scss]: We tend to use actual weights over things like `lighter`, any chance you know what this actual value would be and we can use that, just for clarity/consistency?

Comment (muffinresearch) [reply src/amo/components/LandingPage/styles.scss]: Yep, I'll fix that.

File: src/amo/components/LandingPage/styles.scss (hunk 1 above)
Comment (muffinresearch) [reply src/amo/components/LandingPage/styles.scss]: I think it does, I'll fix that up.

**Pull request merged**